From 6a7daaf6c4a9526068bf6a4595e063e87e0b6a36 Mon Sep 17 00:00:00 2001
From: Matthew Hodgson <matthew@matrix.org>
Date: Fri, 10 Feb 2017 17:21:02 +0000
Subject: [PATCH] remove GA, given it's increasingly useless given Riot/Desktop
 doesn't use it, plus it obviously has privacy implications.  It may get
 re-added as opt-in in future, so we're not sailing entirely blind on how
 folks use the app, or we may switch to Piwik etc instead.

---
 src/vector/index.html | 19 -------------------
 src/vector/index.js   |  1 -
 2 files changed, 20 deletions(-)

diff --git a/src/vector/index.html b/src/vector/index.html
index 3ab667c..e86f299 100644
--- a/src/vector/index.html
+++ b/src/vector/index.html
@@ -62,24 +62,5 @@
         <source src="media/busy.mp3" type="audio/mpeg" />
     </audio>
     <audio id="remoteAudio"/>
-    <script>
-        if (
-                window.location.host === 'www.vector.im' ||
-                window.location.host === 'vector.im' ||
-                window.location.host === 'www.riot.im' ||
-                window.location.host === 'riot.im'
-            ) {
-            (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-            (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-            })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-
-            ga('create', 'UA-54779209-2', 'auto');
-            ga('send', 'pageview', window.location.pathname + window.location.search + window.location.hash);
-        }
-        else {
-            var ga = null;
-        }
-    </script>
   </body>
 </html>
diff --git a/src/vector/index.js b/src/vector/index.js
index d7072d0..0534688 100644
--- a/src/vector/index.js
+++ b/src/vector/index.js
@@ -135,7 +135,6 @@ var onNewScreen = function(screen) {
         var hash = '#/' + screen;
         lastLocationHashSet = hash;
         window.location.hash = hash;
-        if (ga) ga('send', 'pageview', window.location.pathname + window.location.search + window.location.hash);
     }
 }
 
